core-java-collections/src/test/java/com/baeldung/enummap/DummyEnum.java

```diff
@@ -0,0 +1,10 @@
+package com.baeldung.enummap;
+
+public enum DummyEnum {
```

Comment:
Let's add a note that this enum has many values since it is being used for benchmarking.

Reply:
Noted.